### PR TITLE
Issue2 addeslint

### PR DESCRIPTION
**Pull request by ayushjoshi165**
Added Eslint and prettier and airbnb style guide

### CODE REVIEW

File: frontend/src/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (ankitjoshi14) [inline frontend/src/.eslintrc.json]: put the comment to explain why you used this?

File: frontend/src/App.js [hunk 2]
```diff
@@ -2,14 +2,13 @@
 import "./App.css";
 // import UserComponent from "./userComponent";
```
Comment (ankitjoshi14) [inline frontend/src/App.js]: remove any commented code

Comment (ayushjoshi165) [reply frontend/src/App.js]: Removed

**Review by ankitjoshi14 — APPROVED** [review]

**Comment by ankitjoshi14** [comment]
LGTM

**Comment by ayushjoshi165** [comment]
Merged the changes and deleted the issue branch

**Pull request merged**